rosdep/base.yaml

```
@@ -1041,6 +1041,17 @@ fakeroot:
   nixos: [fakeroot]
   rhel: [fakeroot]
   ubuntu: [fakeroot]
+fast-float:
+  arch: [fast_float]
+  debian: [libfast-float-dev]
+  fedora: [fast_float-devel]
+  gentoo: [dev-cpp/fast_float]
+  nixos: [fast-float]
+  osx:
+    homebrew:
+      packages: [fast_float]
+  opensuse: [fast_float]
+  ubuntu: [libfast-float-dev]
```

This package is only available on Noble and newer. Our tooling performs differently when a package has no mapping and when a package can't be found, and the latter is regarded as an error in the rosdep database. We should explicitly state that the package is not expected to exist for older but supported versions of Ubuntu.

```suggestion
  ubuntu:
    '*': [libfast-float-dev]
    focal: null
    jammy: null
```